assets/src/scripts/components/loadingIndicator.js

```diff
@@ -0,0 +1,7 @@
+export const loadingIndicator = function(elem, {showLoadingIndicator, sizeClass}) {
```

Comment:
Can we call this drawLoadingIndicator? That's the pattern in the other modules like date-range-control.js